src/project-conf/index.ts

```diff
@@ -99,3 +121,30 @@ export async function saveProjectConfFile(
     spaces: 2,
   });
 }
+
+/**
+ * Resolves config paths to absolute paths if relative, preserves if already absolute.
+ * Handles both single path string or array of paths.
+ */
+async function resolveConfigPaths(
+  workspaceFolder: Uri,
+  paths?: string | string[]
+): Promise<string | string[]> {
+  if (!paths) return undefined;
+
+  if (Array.isArray(paths)) {
+    return Promise.all(
```

Comment:
Does this function needs to be asynchronous? I believe all inputs and functions used are synchronous.